game/scripts/npc/abilities/death_prophet_silence.txt

```diff
@@ -23,11 +23,11 @@
     // Time
     //-------------------------------------------------------------------------------------------------------------
     "AbilityCooldown"                                     "15 14 13 12 12 12"
-    "AbilityDuration"                                     "3.0 3.5 4.0 4.5 5.0 5.5" //OAA
+    "AbilityDuration"                                     "3.0 4.0 5.0 6.0 6.0 6.0"
```

Comment:
But it is 6 seconds in normal dota. I could see why they nerfed it from 6/8/11 at OAA levels but making an already average spell into an even worse spell doesnt make sense to me.

Reply:
I just compared it with Drow Ranger gust. Same scaling.

Reply:
With max level Octarine Core, DP can perma-silence people.  Also if its the same duration on lvl 5 and 6, there would be no reason to upgrade ability to level 5 or 6. Please revert. Instead of buffing duration: You can make the cd 12 seconds on every level.